curbs/README.md

```diff
@@ -135,7 +133,7 @@ Endpoint: `/curb/policy`
 Method: `GET`  
```

Comment:
Github won't let me comment on the line above this, but looks like a typo, should be `/curbs/`

Reply:
Yes looks like a typo. Will fix in two places.

Reply:
Updated and fixed. https://github.com/openmobilityfoundation/curb-data-specification/blob/feature-release-work-1/curbs/README.md#query-curb-policies